### ISSUE TITLE
[go] request for spec

### ISSUE THREAD

**Issue by mattschrage** [withfig/autocomplete#132]
### CLI Tool

What CLI tool is missing autocompletion in Fig?
`go`

### Useful Links

Please add some links to the docs of that tool or the `--help`text

```
Go is a tool for managing Go source code.

Usage:

	go <command> [arguments]

The commands are:

	bug         start a bug report
	build       compile packages and dependencies
	clean       remove object files and cached files
	doc         show documentation for package or symbol
	env         print Go environment information
	fix         update packages to use new APIs
	fmt         gofmt (reformat) package sources
	generate    generate Go files by processing source
	get         add dependencies to current module and install them
	install     compile and install packages and dependencies
	list        list packages or modules
	mod         module maintenance
	run         compile and run Go program
	test        test packages
	tool        run specified go tool
	version     print Go version
	vet         report likely mistakes in packages
```

Use "go help <command>" for more information about a command.

### Top Workflows

What are the most used commands or workflows in that CLI tool (for you)?

I am not a heavy `go` user (yet)!

**Comment by cstrnt** [comment]
I worked quite a bit with go, l start creating the spec at a basic level 😊

**Comment by bstncartwright** [comment]
@cstrnt Glad to hear! How is that going? I work on Go a lot and am interested in this. I am also happy to help out if desired :)

**Comment by cstrnt** [comment]
@bstncartwright Sorry for the delay! It got merged in with #147 :)
